Fitness/Controllers/GymDetailViewController.swift

```diff
@@ -152,6 +156,16 @@ class GymDetailViewController: UIViewController {
             }
         }
         
+        if !gym.isOpen {
+            closedLabel = UILabel()
+            closedLabel!.font = ._16MontserratSemiBold
```

Comment:
I know the closedLabel exists at this point. But maybe do `let tempClosedLabel = UILabel()` then at the end do `closedLabel = tempClosedLabel`